data_structures/stacks/infix_to_postfix_conversion.py

```diff
@@ -88,14 +88,14 @@ def infix_to_postfix(expression_str: str) -> str:
                 if char_precedence > tos_precedence:
                     stack.push(char)
                     break
-                elif char_precedence == tos_precedence:
-                    if associativity(char) == "RL":
-                        stack.push(char)
-                        break
-                    else:
-                        postfix.append(stack.pop())
-                else:
+                if char_precedence < tos_precedence:
                     postfix.append(stack.pop())
+                    break
```

Comment:
```suggestion
                    continue
```
Sorry, I think this should be a `continue` rather than a `break`, that's my bad 💀

Reply:
Thanks. I was finding the cause of error.